.pipelines/ci.yml

```diff
@@ -35,22 +35,22 @@ stages:
 
     strategy:
       matrix:
-        py311-1163:
+        py312-1171:
           python.version: '3.11'
```

Comment:
the version doesn't match the job name. what's the issue of 3.12 here?

Reply:
[Pipelines - Run 20240229.1 logs (azure.com)](https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=1309882&view=logs&j=498307ac-daeb-53b9-75d4-6986ffc4c71d&t=8eef9f10-c422-50ef-60ef-1c3882233af9): error building onnx wheel because onnx doesn't have python 3.12 package; skipping requirements install and unit test for WindowsPython in latest commit.